### PR TITLE
Story 1 for adding the sort by column

### PR DESCRIPTION
**Pull request by Prashanth-Billa**
I have made the changes, and wanted you to give me a feedback so that I can improve and get back if required.

Thank you

### CODE REVIEW

File: db/model/subject.js [hunk 1]
```diff
@@ -101,6 +101,13 @@ module.exports = function subject(seq, dataTypes) {
       type: dataTypes.STRING(constants.fieldlen.longish),
       allowNull: true,
     },
+    sortBy: {
+      type: dataTypes.STRING(constants.fieldlen.longish),
```
Comment (iamigo) [inline db/model/subject.js]: On second thought, 4096 is too long for this use case... please add a constant to constants.fieldlen called "sortField" with length 254.

File: migrations/20170203233255-addSortByToSubject.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    /*
```
Comment (iamigo) [inline migrations/20170203233255-addSortByToSubject.js]: you need to add your up/down implementations in this file

File: tests/db/model/subject/update.js [hunk 3]
```diff
@@ -218,6 +218,22 @@ describe('db: subject: update: ', () => {
       })
       .catch(done);
     });
```
Comment (iamigo) [inline tests/db/model/subject/update.js]: add a test where sortBy is empty and you update it and then confirm that it's populated
add a test where sortBy is not empty and you update it to set it to null and then confirm that it's null
add a test where sortBy is not empty and you update it to set it to empty string and then confirm that it's empty string

File: tests/db/model/subject/utils.js [hunk 4]
```diff
@@ -21,6 +21,7 @@ const subjectPrototype = {
   helpUrl: 'http://www.bar.com',
   imageUrl: 'http://www.bar.com/foo.jpg',
   isPublished: true,
+  sortBy: ''
```
Comment (iamigo) [inline tests/db/model/subject/utils.js]: Just curious, why are you setting this here?

**Comment by Prashanth-Billa** [comment]
Hey Ian,

All the changes have been made and the codeline is updated. Request you to review and let me know any changes are required.

Thanks
